<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QStackedWidget" name="GuberWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>414</width>
      <height>736</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>550</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userPassword">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>361</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="userEmail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>361</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="enterName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Login Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bookingChoice">
     <widget class="QPushButton" name="carBookingButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Car Booking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planeBookingButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Plane booking</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>371</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Booking Options:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="carBookingPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>-30</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>CAR BOOKING PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmBookingCar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>530</y>
        <width>250</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm booking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="carBackToBooking">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>640</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QQuickWidget" name="CarMapView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>300</width>
        <height>281</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="cyberTruckBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>480</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>CyberTruck</string>
      </property>
     </widget>
     <widget class="QLabel" name="LocationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="planeBookingPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>-30</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>PLANE BOOKING PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmBookingPlane">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>530</y>
        <width>250</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Booking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planeBackToBooking">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>640</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QQuickWidget" name="PlaneMapView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>300</width>
        <height>281</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
      </property>
     </widget>
     <widget class="QLabel" name="locationLabelPlane">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="planeChoices">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>251</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Cool plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ugly plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyber plane (rocket deflection enabled)</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="confirmationPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CONFIRMATION PAGE / TICKET PAGE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
